airbyte-integrations/connectors/source-smaily/source_smaily/smaily.yaml

```diff
@@ -0,0 +1,99 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://{{config['api_subdomain']}}.sendsmaily.net/api"
+    http_method: "GET"
+    authenticator:
+      type: BasicHttpAuthenticator
+      username: "{{config['api_username']}}"
+      password: "{{config['api_password']}}"
+  increment_paginator:
+    type: DefaultPaginator
+    url_base: "*ref(definitions.requester.url_base)"
+    page_size_option:
+      inject_into: "request_parameter"
+      field_name: "limit"
+    pagination_strategy:
+      type: PageIncrement
+      page_size: 250
+    page_token_option:
+      inject_into: "request_parameter"
+      field_name: "page"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
```

Comment:
Why you didn't implement the pagination to all streams? As example campaign endpoint: https://smaily.com/help/api/campaigns-3/list-campaigns/